luda-editor/revert-json-patch/src/lib.rs

```diff
@@ -0,0 +1,194 @@
+#[cfg_attr(test, macro_use)]
```

Comment:
All code this lib.rs is from https://github.com/idubrov/json-patch, I just removed `Move`, `Test`, `Copy` operations, because this crate don't use them for diff.

Reply:
`json-path` is licensed under `Apache License, Version 2.0`.
Maybe we should post this comment at `namseent/NOTICE`?

Reply:
Actually it's under Apache or Mit licence. 
![image](https://user-images.githubusercontent.com/3580430/193766047-c128848f-e03a-490c-b138-261d74e85371.png)